packages/holocron/src/loadModule.web.js

```diff
@@ -53,14 +53,29 @@ function createScript({ url, integrity, onLoad = noop }) {
   return listener;
 }
 
-function loadModuleFallbackExternals(moduleName) {
-  const fallbacks = getUnregisteredRequiredExternals(moduleName);
+const loadingFallbacks = new Map();
+
+async function loadModuleFallbackExternals(moduleName, fallbacks) {
   const baseUrl = getModuleMap().getIn(['modules', moduleName, 'baseUrl']);
 
-  return Promise.all(fallbacks.map(({ name, browserIntegrity }) => createScript({
-    url: `${baseUrl}${name}.browser.js`,
-    integrity: browserIntegrity,
-  })));
+  await Promise.all(fallbacks.map(async ({ name, browserIntegrity }) => {
+    if (loadingFallbacks.has(browserIntegrity)) {
```

Comment:
Some thoughts on using browserIntegrity: 
could updates to the bundler alter the build output resulting in the same package ending with different integrity values? If this is the case would `name-version` be a more reliable key ?

Reply:
yeah could happen, and i think `name-version` or something similar would make it easier to debug and less prune to mismatches